src/osp/Active/SysNewton.cpp

```diff
@@ -200,6 +194,9 @@ void SysNewton::create_body(ActiveEnt entity)
 
         // Set inertia and mass
         NewtonBodySetMassMatrix(entBody.m_body, entBody.m_bodyData.m_mass, 1, 1, 1);
+
+        Vector3 tmpOset{0.0f, 0.0f, 0.0f};
+        NewtonBodySetCentreOfMass(entBody.m_body, tmpOset.data());
```

Comment:
NewtonBodySetCentreOfMass(entBody.m_body, Vector3{0.0f, 0.0f, 0.0f}.data());

Reply:
This is temporary code used to test something z-adams was trying, removed in the latest commit.